template/src/features/counter.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const slice = createSlice({
+  name: 'counter',
+  initialState: {
+    value: 0,
+  },
+  reducers: {
+    increment: state => {
+      state.value += 1;
+    },
+    decrement: state => {
+      state.value -= 1;
+    },
+  },
+});
+
+const selectCount = state => state.counter.value;
+
+export const selectors = {
```

Comment:
I don't think exporting the selector inside of an object helps here.  Just do `export const selectCount =`.

For that matter, does the selector really buy us anything useful in terms of the example?  It at least shows the idea of a selector, but it also doesn't show use of Reselect.

Reply:
I always like using selectors even for small things, since it hides the data structure from the consumers and therefore is easier to change. IMO the data structure of the state should be an implementation detail of the slice. 

Up to you though.